ecl/hql/hqlexpr.cpp

```diff
@@ -15061,6 +15062,10 @@ extern HQL_API bool hasUnknownTransform(IHqlExpression * expr)
         if (expr->hasAttribute(mergeTransformAtom))
             return true;
         break;
+    case no_call:
```

Comment:
I'm not sure this is the correct place to put this call - it may cause problems elsewhere.